PublicMapping/districtbuilder
# Add Support Menu to landing and project page

## Overview
Adds a support menu that allows users to access documentation and/or contact us if they get stuck. The original plan was to add in a chat window, but we will do this post-launch once we've purchased a chat service.

### Checklist


### Demo
<img width="261" alt="Screen Shot 2020-09-18 at 8 06 56 PM" src="https://user-images.githubusercontent.com/5672295/93635633-8a872e80-f9ea-11ea-8a92-92c7c61503fb.png">

<img width="1231" alt="Screen Shot 2020-09-18 at 8 05 12 PM" src="https://user-images.githubusercontent.com/5672295/93635566-6d526000-f9ea-11ea-8017-2d9d5cb0dafd.png">

<img width="243" alt="Screen Shot 2020-09-18 at 8 06 50 PM" src="https://user-images.githubusercontent.com/5672295/93635651-9115a600-f9ea-11ea-8e11-fc9f95f8c587.png">

<img width="1229" alt="Screen Shot 2020-09-18 at 8 06 27 PM" src="https://user-images.githubusercontent.com/5672295/93635595-793e2200-f9ea-11ea-8c2f-10aaa39306cd.png">


### Notes
- This actually ended up borrowing a lot from the componentized menu that @kshepard made in a different PR, ty ty 🙏 
- One thing that came up was inverting the trigger button styles depending on where it is located is important. I did this with a separate file that takes a boolean prop and spits out some `sx` styles, but there may be a better way to do this. Def let me know if anything about it could be better.
- Similarly, maybe it's overkill to keep the url's as separate variables, rather than directly placed in the `href` property? I wasn't sure what felt cleaner and am open to feedback.
- Both links open in a new tab. I wasn't sure if people could lose their work if they leave the current page + both actions seem like a departure from the task at hand. Let me know if this isn't desired.

## Testing Instructions
- `git pull`
- Login. On the Maps landing page, interact with the new "Support" dropdown.
- Click "Getting Started Guide" to see the GitHub wiki page open in a new tab. 
- Now click "Contact us" to see your mail client open up an email addressed to `support@districtbuilder.org` in a new tab.
- Enter a map.
- On the internal project page, interact with the new "Support" dropdown.
- Click "Getting Started Guide" to see the GitHub wiki page open in a new tab. 
- Now click "Contact us" to see your mail client open up an email addressed to `support@districtbuilder.org` in a new tab.

Closes #394
